client/gutenberg/extensions/hello-dolly/hello-block.js

```diff
@@ -0,0 +1,38 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import Badge from 'components/badge';
+import Button from 'components/button';
+import Notice from 'components/notice';
+
+import 'components/badge/style.scss';
+import 'components/button/style.scss';
+import 'components/notice/style.scss';
+
+const { InspectorControls } = wp.editor;
+
+wp.blocks.registerBlockType( 'calypsoberg/hello-dolly', {
+	title: 'Hello Dolly',
+	icon: 'format-audio',
+	category: 'layout',
+	edit: ( { isSelected } ) => (
+		<>
```

Comment:
`i18n-calypso` doesn't understand `<>` JSX yet, breaks the translation string parsing.

The changes are in master (https://github.com/Automattic/i18n-calypso/pull/55), but a version needs to be released and Calypso upgraded.

Reply:
it can't _translate_ that call into `<React.Fragment />` ? 🙃

Reply:
We should should use `React.Fragment` (`wp.element.Fragment`??? :confounded: or if all else fails, `<div />`) for now to unblock this.

Reply:
Array should also be a valid option (remember that from two months ago? 😆)

I think we could unblock `<>` syntax rather quickly:
- https://github.com/Automattic/i18n-calypso/pull/64
- https://github.com/Automattic/gp-localci-client/pull/12
- Update to `i18n-calypso@2.0.0` in Calypso.